Formula/sonarqube.rb

```diff
@@ -18,12 +18,33 @@ class Sonarqube < Formula
   end
 
   # sonarqube ships pre-built x86_64 binaries
-  depends_on arch: :x86_64
   depends_on "openjdk@11"
 
   conflicts_with "sonarqube-lts", because: "both install the same binaries"
 
+  resource "java-service-wrapper" do
+    url "https://download.tanukisoftware.com/wrapper/3.5.46/wrapper-macosx-arm-64-3.5.46.tar.gz"
```

Comment:
We don't want to download a precompiled binary. This needs to be built from source (as I believe it was in the last revision)

Reply:
The last revision is already using precompiled binaries and precompiled jar files from https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-9.1.0.47736.zip.

What do you think about building a keg only formula for Java Service Wrapper ?

Reply:
Sounds good to me